lib/runFromNetwork.dart

```diff
@@ -152,20 +179,32 @@ class _RunFromNetwork extends State<RunFromNetwork>
     }
 
     maybeReload();
-    timer = Timer.periodic(
+    /*timer = Timer.periodic(
```

Comment:
Hot reload is driven by polling. If the timer isn't initialized, then the content will never reload. Not initializing the timer is also causing a crash trying to call `timer.cancel` in `dispose`.

Reply:
You are absolutely right, I just forgot to merge my branches correctly. Should be fixed in the new commit